.github/workflows/CI.yml

```diff
@@ -92,12 +92,78 @@ jobs:
       if: ${{ contains(matrix.build, 'cmake') }}
       run: cmake --install ${{ env.BUILD_DIR }}
 
-  intel-build:
+  intel-build-llvm:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
+        fc: [ifx]
+        cc: [icx]
+        cxx: [icpx]
+    env:
+      FC: ${{ matrix.fc }}
+      CC: ${{ matrix.cc }}
+      CXX: ${{ matrix.cxx }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```

Comment:
can we shift to the v4 , there had been some bugs in 'v2'.

Reply:
Thank you. I applied this change.